packages/next/README.md

```diff
@@ -486,7 +486,7 @@ class HelloUA extends React.Component {
 export default HelloUA
 ```
 
-Notice that to load data when the page loads, we use `getInitialProps` which is an [`async`](https://zeit.co/blog/async-and-await) static method. It can asynchronously fetch anything that resolves to a JavaScript plain `Object`, which populates `props`.
+Notice that to load data when the page loads, we use `getInitialProps` which is an [`async`](https://zeit.co/blog/async-and-await) method. It can asynchronously fetch anything that resolves to a JavaScript plain `Object`, which populates `props`.
```

Comment:
```suggestion
Notice that to load data when the page loads, we use `getInitialProps` which is an [`async`](https://zeit.co/blog/async-and-await) static method. It can asynchronously fetch anything that resolves to a JavaScript plain `Object`, which populates `props`.
```

It's still a static method, even when not set using the `static` keyword.